### PR TITLE
Add correct Identity Hub ServiceEndpoint format

### PR DESCRIPTION
**Pull request by csuwildcat**
***
<a href="https://pr-preview.s3.amazonaws.com/csuwildcat/did-spec/pull/230.html" title="Last updated on Jun 25, 2019, 1:38 PM UTC (7bc736d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/did-spec/230/31ae2ee...csuwildcat:7bc736d.html" title="Last updated on Jun 25, 2019, 1:38 PM UTC (7bc736d)">Diff</a>

### CODE REVIEW

**Comment by peacekeeper** [comment]
@csuwildcat just one question: We had a discussion a while ago about allowing DIDs as values for `serviceEndpoint` (see https://github.com/w3c-ccg/did-resolution/issues/7). Is this still the same idea, only that a new term (`instances`) is used instead?

File: index.html [hunk 1]
```diff
@@ -1452,8 +1452,13 @@ <h2>
     "serviceEndpoint": "https://agent.example.com/8377464"
   }, {
     "id": "did:example:123456789abcdefghi#hub",
-    "type": "HubService",
-    "serviceEndpoint": "https://hub.example.com/.identity/did:example:0123456789abcdef/"
+    "type": "IdentityHub",
+    "publicKey": "did:example:123456789abcdefghi#key-1",
+    "serviceEndpoint": {
+      "@context": "https://schema.identity.foundation/hub",
+      "@type": "UserHubEndpoint",
```
Comment (brentzundel) [inline index.html]:
```suggestion
      "type": "UserHubEndpoint",
```

**Comment by msporny** [comment]
The group agreed to merge this on the 2019-08-01 call with a change: change @type to change it to type.

**Comment by rhiaro** [comment]
Merged with `type` fix via https://github.com/w3c-ccg/did-spec/pull/245

**Pull request closed without merging**